Zend/zend_inheritance.c

```diff
@@ -1566,6 +1566,10 @@ static void do_inherit_property(zend_property_info *parent_info, zend_string *ke
 						ZSTR_VAL(key),
 						ZSTR_VAL(parent_info->ce->name));
 			}
+
+			if (child_info->flags & ZEND_ACC_OVERRIDE) {
+				child_info->flags &= ~ZEND_ACC_OVERRIDE;
+			}
```

Comment:
This can just unconditionally try to remove the flag:

```suggestion
			child_info->flags &= ~ZEND_ACC_OVERRIDE;
```